src/tabs.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+/**
+ * @author Martin Nicholson <martin@taotesting.com>
+ */
+/**
+ * @example
+ * var instance = tabs({
+ *   renderTo: $container,
+ *   tabs: [
+ *     { label: 'TAO Local', name: 'local-delivery' },
+ *     { label: 'TAO Remote', name: 'remote-delivery' },
+ *     { label: 'LTI-based', name: 'lti-delivery', disabled: true }
+ *   ],
+ *   activeTabIndex: 1
+ * });
+ *
+ */
+import $ from 'jquery';
+import _ from 'lodash';
+import component from 'ui/component';
+import tabsTpl from 'ui/tabs/tpl/tabs';
+import 'ui/tabs/css/tabs.css';
+
+const ns = 'tabs';
+
+/**
+ * Default config
+ * @param {Integer} activeTabIndex - the index of the tab to start on
+ * @param {Boolean} showHideTargets - if true, no need to wire up tabs to panels, it's automatic
+ * @param {String} targetDataAttr - the data attribute suffix used to connect tabs to panels
```

Comment:
I'd suggest calling this by default tab-content, panel feels a bit out of the context.

Reply:
Done. It's now `data-tab-name` on the button and `data-tab-content` on the content element. Non-configurable.